go.mod

```diff
@@ -1,10 +1,11 @@
 module github.com/app-sre/vault-manager
 
-go 1.17
+go 1.18
```

Comment:
Feel free to use 1.19 here already. There should be no side effects from doing so unless you have seen something?